pyvistaqt/plotting.py

```diff
@@ -302,7 +314,8 @@ def dragEnterEvent(self, event):  # pylint: disable=invalid-name,no-self-use
         except IOError as exception:  # pragma: no cover
             warnings.warn("Exception when dropping files: %s" % str(exception))
 
-    def dropEvent(self, event):  # pylint: disable=invalid-name
+    # pylint: disable=invalid-name,useless-return
+    def dropEvent(self, event: QtCore.QEvent) -> None:
```

Comment:
Why bother adding the `-> None`? Can it be omittted and the `return None` removed / not added? Or is it required by mypy to always have the return values even when None?

Reply:
Thank you for comment. As you say mypy always have the return values even when None.